packages/npm/send/example/src/App.tsx

```diff
@@ -15,52 +23,110 @@ export const serializeError = (error: unknown) => {
   }
 }
 
-export default function App() {
-  const [result, setResult] = React.useState<string>('waiting...')
+const App = () => {
+  const [data, setData] = useState<{
+    trackName: string
+    artistName: string
+    artworkUrl100: string
+    description: string
+    version: string
+    screenshotUrls: string[]
+  } | null>()
```

Comment:
```suggestion
  } | null>(null)
```
Otherwise, won't this default to undefined?